lib/config_file.js

```diff
@@ -3,6 +3,7 @@ var readFileSync = require('fs').readFileSync;
 
 var configFiles = [ '.kibana-plugin-helpers.json', '.kibana-plugin-helpers.dev.json' ];
 var configCache = {};
+var KIBANA_ROOT_OVERRIDE = process.env.KIBANA_ROOT ? resolve(process.env.KIBANA_ROOT) : null;
```

Comment:
I'm not opposed, I was just curious what the impetus for this PR was. CI seems like a legit reason.